galley/pkg/config/processing/snapshotter/scope.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snapshotter
+
+import (
+	"istio.io/pkg/log"
+)
+
+var scope = log.RegisterScope("processing", "", 0)
```

Comment:
It isn't obvious that `scope` is a logging scope.  Maybe call it `logScope` or move it to a `log` subpackage?

Reply:
If it wasn't in a `log` package, then we probably should have used `logScope` throughout instead. Deviating from our previous pattern here is of very low cost.

Reply:
It is everywhere though, not just Galley code and not in collab-galley only. We simply started that way. If we're going to make the change, I'd rather have us do a sweeping change specifically for that and not be inconsistent.

Reply:
ok that's reasonable.